Example/Colander/InteractiveDataSource/InteractiveDataSourceViewController.swift

```diff
@@ -25,37 +25,18 @@ class InteractiveDataSourceViewController: UIViewController, CalendarViewDataSou
 
     let calendarView = CalendarView()
 
-    lazy var startDateTextField: UITextField = {
-        let textField = UITextField()
-        textField.inputAccessoryView = self.toolbar
-        textField.inputView = self.datePicker
-        textField.textColor = .blue
-        textField.delegate = self
-        return textField
-    }()
-    
-    lazy var endDateTextField: UITextField = {
-        let textField = UITextField()
-        textField.inputAccessoryView = self.toolbar
-        textField.inputView = self.datePicker
-        textField.textColor = .blue
-        textField.delegate = self
-        return textField
-    }()
-
-    lazy var datePicker: UIDatePicker = {
+    lazy var startDatePicker: UIDatePicker = {
```

Comment:
The new date pickers mean there doesn't have to be a text field to present the value.